docker-compose.yml

```diff
@@ -1,9 +1,15 @@
-version: "3"
-
 services:
-  frank:
+  frankframework:
+    image: frankframework/frankframework:latest
+    ports:
+      - "8080:8080"
+    volumes: &frank-volumes
+      - ./frontend/src/main/frontend/cypress/Frank/src/main/:/opt/frank/
+    environment:
+      management.gateway.inbound.class: "org.frankframework.management.gateway.HazelcastInboundGateway"
+  frank-flow:
     image: frank-flow
     build: .
     ports:
-      - "8080:8080"
-    restart: unless-stopped
+      - "8081:8080"
+    volumes: *frank-volumes
```

Comment:
What does the `*` do in front of the volume name?
I can't find anything about it in their reference page

Reply:
You are in for a treat! 😀 
This is called a YAML Alias. It references the volumes I defined at the frankframework server, there it is declared as YAML Anchor.
Using this and merging compose-files can make Docker Compose really powerful and flexible.

If you want to read more about this and YAML overrides you can read this: https://www.linode.com/docs/guides/yaml-anchors-aliases-overrides-extensions/

Reply:
Wow this is nice!